src/Ontology/Address.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Address.h"
+
+#include "Hash.h"
+#include "HexCoding.h"
+#include <TrezorCrypto/sha2.h>
+#include <TrezorCrypto/base58.h>
+#include <TrezorCrypto/ripemd160.h>
+#include <iostream>
+#include <stdexcept>
+
+using namespace TW::Ontology;
+
+std::vector<uint8_t> Address::sha256(std::vector<uint8_t> &data) {
```

Comment:
Why not use `Hash::sha256`?

Reply:
sorry, I'm not familiar with the functions in library. using `Hash::sha256` would be more suitable, I will correct it.

Reply:
https://github.com/TrustWallet/wallet-core/pull/103/commits/0f8bf1c78da5f7cd69389bca0146a29aea5c9586